anchore_engine/analyzers/modules/32_python_packages.py

```diff
@@ -25,6 +25,9 @@
 unpackdir = config['dirs']['unpackdir']
 squashtar = os.path.join(unpackdir, "squashed.tar")
 
+# known modules that generate false positives since they are part of core Python (2) runtime, for which vulnerabilities are handled separately
+omit_pymods = ['Python']
```

Comment:
Can you add a 'fixes...' to the PR merge request for the issues this resolves? I think it will be good to call out in the changelog.

Reply:
done!